plugin-guide.md

```diff
@@ -42,11 +54,31 @@ The plugins can be associated with targets in the server configuration.
 Consumable resources _may_ request that submitters provide information or
```

Comment:
Is there a better definition elsewhere in the docs on what a consumable resource actually is? To me this sounds like 'RAM' but that doesn't seem to mesh with the rest of the documentation here on it